frontend/src/pages/ExperimentList.test.tsx

```diff
@@ -70,14 +71,22 @@ describe('ExperimentList', () => {
     formatDateStringSpy.mockImplementation(() => '1/2/2019, 12:34:56 PM');
   });
 
+  afterEach(async () => {
+    // unmount() should be called before resetAllMocks() in case any part of the unmount life cycle
+    // depends on mocks/spies
+    if (tree) {
```

Comment:
You don't need the check. I don't think anything happens if the tree is unmounted.

Reply:
Why the `await`?

Reply:
I was seeing errors trying to unmount the tree if it was already unmounted (or had never been mounted).

It seems like the `await` probably isn't needed. We do it in many other tests though. I'll go through and clean that up